src/assets/main.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: 400;
+}
+img {
+  max-width: 100%;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  font-family: 'Playfair Display', serif;
```

Comment:
see comments above, use a variable for this

Reply:
compiled files shouldn't be on the repo, same for the "builds" and things like that, if you have to keep de assets folder there is no problem to keep it empty "if you dont know why you can ask me" but if dont please remove it

Reply:
please remove this file and the other compiled files